lib/core/src/display/shape/text/text_field.rs

```diff
@@ -84,6 +84,21 @@ shared! { TextField
             self.display_object.set_position(position);
         }
 
+        /// Get position of this TextField.
+        pub fn position(&self) -> Vector3<f32> {
+            self.display_object.position()
+        }
+
+        /// Set size.
+        pub fn set_size(&mut self, size:Vector2<f32>) {
+            self.properties.size = size;
```

Comment:
To make it work, you should also recreate the Sprites structure (or no text will display after extending this component)
And that's not a simple change, you could do 
```rust
self.rendered = TextFieldSprites::new(world,&properties);
```
But it required world reference. It should be done better, but I think I create a new task for it.

Reply:
I tried this workaround, but it wasn't working. It was panicking for some reason. Would you mind providing a better solution for it in your task?

Reply:
If this does not work and panics, lets remove this `set_size` method, and just create text_field with initial screen size (without resizing). I'll create a task for changing properties of TextField.